hax/hax/handler.py

```diff
@@ -149,10 +149,8 @@ def update_process_failure(self, planner: WorkPlanner,
             for state in ha_states:
                 if state.fid.container == ObjT.PROCESS.value:
                     is_proc_local = self.consul.is_proc_local(state.fid)
-                    current_status = state.status
-                    if state.status in (ObjHealth.OFFLINE, ObjHealth.FAILED):
```

Comment:
Question: Do we not need this check anymore?

Reply:
Yep not needed anymore.